src/core/settings.c

```diff
@@ -867,6 +867,106 @@ QuicSettingsLoad(
         Settings->VersionNegotiationExtEnabled = !!Value;
     }
 
+    if (!Settings->IsSet.VersionSettings) {
+        uint32_t OfferedVersionsSize = 0, AcceptableVersionsSize = 0, FullyDeployedVersionsSize = 0;
+        if (QUIC_SUCCEEDED(
+                CxPlatStorageReadValue(
+                    Storage,
+                    QUIC_SETTING_OFFERED_VERSIONS,
+                    (uint8_t*)NULL,
+                    &OfferedVersionsSize)) &&
```

Comment:
nit: We follow pattern of aligning `QUIC_SUCCEEDED(` with the function on the next line
```suggestion
        if (QUIC_SUCCEEDED(
            CxPlatStorageReadValue(
                Storage,
                QUIC_SETTING_OFFERED_VERSIONS,
                (uint8_t*)NULL,
                &OfferedVersionsSize)) &&
```
Could you please update this code and below?